tests-e2e/ocw-ci-test-course/resource-list.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from "@playwright/test"
+import { CoursePage } from "../util"
+
+test("Resource list shows correct thumbnails and aria labels for different types of files", async ({
+  page
+}) => {
+  const course = new CoursePage(page, "course")
+  await course.goto("/lists/a-resource-list")
+  const resourceLocators = await page
+    .locator(".resource-list-item > .row")
+    .all()
+
+  for (const resource of resourceLocators.values()) {
+    const thumbnailLocator = resource.locator("img.resource-thumbnail")
+
+    expect(thumbnailLocator).toBeVisible()
+
+    const title = await resource.locator(".resource-list-title").textContent()
+    const src = await thumbnailLocator.getAttribute("src")
```

Comment:
See Playwright's [recommended locators](https://playwright.dev/docs/locators#quick-guide)

Here, it would be better to do `page.getByRole("img", { name })`
- less fragile—doesn't dependend on CSS classes that might change
- essentially is an assertion that our elements have appropriate ARIA roles (usually inherited from the underling html element, like `<img />` having `"img"` role).